src/event_request_bot.py

```diff
@@ -0,0 +1,123 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+from selenium.webdriver.common.keys import Keys
+import time
+import getpass
+from user import User
+from event import Event
+
+# WEBDRIVER_PATH = '..\\assets\\geckodriver.exe'  # Windows driver
+WEBDRIVER_PATH = '../assets/geckodriver'  # Mac driver
+# USER_NAME = ''
+# PASSWORD = ''
+#
+# EVENT_NAME = 'Fake Event'
+# EVENT_DESC = 'This is a totally testable description'
+# START_DATE = '01 Apr 2019'
+# START_TIME = '01:15 PM'
+# # END_DATE = '01 Apr 2019'
+# END_TIME = '02:15 PM'
+# EVENT_LOCATION = 'Drexel Park'
+# EST_ATTENDANCE = '20'
+# SAFAC_ACC_NUM = '170178'
+# EST_COST = '$0'
+# PHONE_NUMBER = '2672748265'
+CREATE_LINK = 'https://dragonlink.drexel.edu/submitter/organization/'+'FISDU'+'/eventsubmission/create'

```

Comment:
Move the CREATE_LINK into the request function and utilize the `User.org` attribute.

Reply:
Fixed in [5bd6f13](https://github.com/AbirRazzak/event-request-bot/pull/7/commits/5bd6f13ca9e6fc52b99f17de70fcd102307cd43b)